src/views/token-swap/components/quote/quote.view.jsx

```diff
@@ -44,19 +44,6 @@ function Quote({
             </p>
           )}
         </div>
-        <div className={classes.rewardCell}>
```

Comment:
this is from token swap, so it should stay. It's' a different kind of reward

Reply:
Thx! Reverted!